spec/controllers/api/venues_controller_spec.rb

```diff
@@ -0,0 +1,69 @@
+require 'rails_helper'
+require 'pry'
+
+RSpec.describe Api::V1::VenuesController, type: :controller do
+  let(:organizer) { create(:user, role: :organizer) }
+
+  describe 'GET #show' do
+    let(:venue_attributes) { %w[id address description] }
+
+    context 'when valid' do
+      let(:venue) { create(:venue) }
+      it 'returns json response with venue' do
+        get :show, params: { id: venue.id }
+        json_response
```

Comment:
?

Reply:
@mesmerze recommended me this, as part of DRY principle.

Reply:
remove

Reply:
done